runelite-api/src/main/java/net/runelite/api/events/WidgetMenuOptionClicked.java

```diff
@@ -28,7 +28,7 @@
 import net.runelite.api.widgets.WidgetInfo;
 
 /**
- * An event where an option has been clicked in a {@link net.runelite.api.widgets.Widget}s menu.
+ * A MenuManager widget menu was clicked. This event is NOT fired for non-MenuManager menu options
```

Comment:
Either add link to MenuManager here or do not mention it if it's not reachable from here (and as it is in client, it is not reachable).

Reply:
I can't link to MenuManager because this event is in the wrong module

Reply:
Makes sense. I guess we can move it in separate PR.